docs/developer-docs/latest/setup-deployment-guides/configurations/optional/api.md

```diff
@@ -0,0 +1,34 @@
+---
+title: API configuration - Strapi Developer Documentation
+description:
+---
+
+<!-- TODO: update SEO -->
+
+# API configuration
+
+General settings for API calls can be set in the `./config/api.js` file:
+
+| Property                      | Description                                                                                                 | Type         | Default |
+| ----------------------------- | ----------------------------------------------------------------------------------------------------------- | ------------ | ------- |
+| `responses`                   | Global API response configuration                                                                           | Object       | -       |
+| `responses.privateAttributes` | Set of globally defined attributes to be treated as private.                                                | String array | `[]`    |
+| `rest`                        | REST API configuration                                                                                      | Object       | -       |
+| `rest.defaultLimit`           | Default `limit` parameter used in API calls                                                      | Integer      | `100`   |
+| `rest.maxLimit`               | Maximum allowed number that can be requested as `limit`.<br/><br/>Defaults to `null`, which fetches all results | Integer      | `null`  |
```

Comment:
@alexandrebodin I believe there needs to be a config value for the REST api prefix here also correct?